data/helpers.d/network

```diff
@@ -0,0 +1,9 @@
+# Check availability of a web path
+#
+# example: ynh_path_validity $domain$path
+#
+# usage: ynh_path_validity $domain_and_path
+# | arg: domain_and_path - complete path to check
+ynh_path_validity () {
+	sudo yunohost app checkurl $1 -a $app
```

Comment:
it would be better to not call the moulinette at all, and instead rewrite it in pure bash.